common/discordutil.js

```diff
@@ -101,6 +97,14 @@ module.exports = {
       embed.addFields(fields);
     }
 
+    return embed;
+  },
+
+  async sendGiveApplesEmbed(channel, title, description, fields = [], thumbnail = '') {
+    if (!channel) {
+      return;
+    }
+    const embed = this.createScheduledJobEmbed('giveApples', title, description, '#5445ff', fields, thumbnail);
```

Comment:
Testing again, and the this here gives me an error when running the apple schedule. 😢🥺 Maybe we can move this function to the giveApples file? 🤔